pkg/observable/channel/replay_test.go

```diff
@@ -0,0 +1,87 @@
+package channel_test
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+
+	"pocket/pkg/observable/channel"
+)
+
+func TestReplayObservable(t *testing.T) {
+	var (
+		n, values   = 3, []int{1, 2, 3, 4, 5}
```

Comment:
NIT: more semantic name for n (e.g. replayBufferSize)

Reply:
`n` is the value that is passed to `#Last()` as the `n` arg. I've updated it to `lastN`.